src/Symfony/Bridge/PhpUnit/Legacy/ExpectDeprecationTrait.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Legacy;
+
+/**
+ * @internal use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait instead.
+ */
+trait ExpectDeprecationTrait
+{
+    /**
+     * @param string $message
+     */
+    public function expectDeprecation(): void
```

Comment:
aren't we allowed to add an optional argument?
`expectDeprecation(string $message = null): void`?

Reply:
We can but it brings wrong autocompletion. Better have a correct PHPDoc than a wrong real param I guess?